app/src/main/java/org/mozilla/fenix/search/SearchDialogFragment.kt

```diff
@@ -271,6 +269,40 @@ class SearchDialogFragment : AppCompatDialogFragment(), UserInteractionHandler {
             requireContext().settings().setCameraPermissionNeededState = false
         }
 
+        // To increase touch area of the toggle buttons, without altering actual view bounds
```

Comment:
We probably want this comment before the start of the `search_engines_shortcut_layout.post` block.

Otherwise, I wanted to note that there are quite a few implementations of `dpToPx()` in Fenix and AC. I am wondering if anyone of them might be doing the same thing as what you want to avoid adding another `dpToPx()`. 😱